### ISSUE TITLE
Feature Request: Guest access

### ISSUE THREAD

**Issue by Kyoshiro-san** [igniterealtime/Openfire-Meetings#61]
Hi !

I haven't found a previous question like this: would it be possible to somehow allow guest access to a meeting ?

Thank you very much !
Kinds Regards.

**Comment by deleolajide** [comment]
Yes, enable anonymous access from openfire and your guests can attend meetings. 
![image](https://user-images.githubusercontent.com/110731/37778816-c94c2ade-2de2-11e8-8a9c-39d0344e2f50.png)

To secure your meeting,  add a password after the meeting starts

![image](https://user-images.githubusercontent.com/110731/37778996-32ff215c-2de3-11e8-940e-6cb06de5b980.png)

**Comment by Kyoshiro-san** [comment]
Thank you for your answer @deleolajide 

We would like to keep our server private by default, and allow guest access only or people having the link. Having guest access only via ofmeet web access could be a solution, but I can't find such an option.

Is there a solution maybe with default permissions ?

Thank you very much.
Kind regards.

**Comment by marclaporte** [comment]
In one of my use cases: Only registered users should be able to start a WebRTC room. And they should be able to set it to be open to anonymous, or password protected (so accessible without a username and password)
